docs/source/guide/benchmarks.md

```diff
@@ -0,0 +1,133 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.4
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Benchmarks
```

Comment:
```suggestion
# Benchmarking Circuits
```

Could you please relabel the file as `docs/source/guide/benchmarking_circuits.md`